src/client.ts

```diff
@@ -1125,6 +1131,13 @@ export class StreamChat<
     }
   };
 
+  _retryInterval = () => {
+    // try to reconnect in 0.25-25 seconds (random to spread out the load from failures)
+    const max = Math.min(500 + this.consecutiveFailures * 2000, 25000);
+    const min = Math.min(Math.max(250, (this.consecutiveFailures - 1) * 2000), 25000);
+    return Math.floor(Math.random() * (max - min) + min);
+  };
```

Comment:
I think we have a similar implementation for ws class, maybe refactor this func into the util file?

Reply:
Yes it is the same.

Reply:
refactored.